server.js

```diff
@@ -0,0 +1,11 @@
+const express=require("express");
+const app=express();
+const port=3000
+
+app.get("/ping",(req,res)=>{
+    res.send("Weirdy Foody");
+})
+
+app.listen(port, () => {
+    console.log(`server is running on PORT: ${port}`);
+});
```

Comment:
Add a newline at the end of the file. This is a common coding convention that helps with file processing in various tools and environments. [medium]